templates/basic/TypeScript/discordjs/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "discord-bot-ts",
+  "description": "This project was created with **create-discord-app**!",
+  "version": "1.0.0",
+  "main": "build/index.js",
+  "scripts": {
+    "dev": "nodemon src/index.ts",
+    "build": "set NODE_ENV=build && tsc && node build/index.js"
```

Comment:
Don't start the process on `build`, only run `tsc` and `cross-env NODE_ENV=production node build/index` as `start` script